src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    const action = actions[i].type;
```

Comment:
Use action object instead of action type directly. Replace `const action = actions[i].type;` with `const action = actions[i];`